octoprint_Spoolman/static/js/Spoolman_sidebar.js

```diff
@@ -47,65 +46,67 @@ $(() => {
             self.templateData.loadingError(undefined);
             self.templateData.isLoadingData(true);
 
-            const spoolmanSpoolsResult = await pluginSpoolmanApi.getSpoolmanSpools();
+            try {
```

Comment:
Could you please explain why adding this try catch was necessary? Usually, any API calls should result in a non-throwing call result, but I might have missed some edge cases in the generic code. If that's the case, it would be better to fix that over there, but I need to understand the situation better to determine whether that's the case.